### DISCUSSION TITLE
cdnjs package

### DISCUSSION THREAD

**Discussion by zdynamics** [pdfme/pdfme#125] [Ideas]
It would be great to have a package in unpkg.com or cdnjs (like jsPDF or pdf.lib) to use basic functionalities such as rendering a PDF from a template, only modifying some text with JavaScript

**Comment by hand-dot** [comment]
Hi @zdynamics 
Thank you for the suggestion.
But I don't use unpkg.com or cdnjs.
So, I will not support it.
I'll publish only the ESM version on the next version.

Of course, if there are a lot of needs, I'll support it.
If some one agrees with this suggestion, please comment on this thread.
